### ISSUE TITLE
Create a system tray with the backup running.

### ISSUE THREAD

**Issue by ColonyCreative** [linuxmint/timeshift#213]
**Describe the solution you'd like**
The goal would be, as soon as Timeshift starts a backup, a system tray should appear in GNOME, where I can see the current progress in percent by clicking on it.

**Additional context**
I am a programmer myself and would take care of it. The only question is whether I am the only one with the desire, or whether several people have this desire?

**Comment by yegorich** [comment]
Do you mean somethig like [this](https://github.com/linuxmint/timeshift/blob/master/src/Gtk/EstimateBox.vala#L112)? Though, it would work only in Cinamon and it doesn't work in Linux Mint for me :-(

**Comment by ColonyCreative** [comment]
> Do you mean somethig like [this](https://github.com/linuxmint/timeshift/blob/master/src/Gtk/EstimateBox.vala#L112)? Though, it would work only in Cinamon and it doesn't work in Linux Mint for me :-(

It does not have to be a progress bar, important would be once the status and the percentage.
I would implement it, but if no one needs it, I don't go to the extra trouble.

**Comment by yegorich** [comment]
Such a feature would be very useful.

**Comment by bonnebulle** [comment]
yep, an easy way to know when Timeshift is working (making my CPU very hot)
A systray / panel icon, a notification...
- A notification should be easy to program with a script (triggered after work is done)
- A systray icon (changing when Timeshift is working) + options / menu... it is another work to develop and maintain !
.. and a good feature
thanks

**Comment by ygerlach** [comment]
At least on my system with linuxmint/Cinnamon timeshift has a systray icon, showing a gear-icon when a snapshot operation is running. But it has no menu to interact with it in anyway.
And i believe i have also seen some code related to trigger a notification.

I would also like to improve on the systray icon, but i have not yet found the code responsible for it.
#295 and #469 seems related.
